pskb_website/static/js/editor_utils.js

```diff
@@ -317,6 +317,7 @@ function save(sha, path, secondary_repo, action_url) {
     var stacks = document.createElement("input");
     stacks.name = "stacks";
     stacks.value = stacks_select.value;
+    stacks.required = "required";
```

Comment:
What does this do?  I tested it and selecting a stack is not required.  We don't want to require a stack since we have a finite list, so just curious what this line does.